src/routes/docs/sdks/+page.markdoc

```diff
@@ -297,6 +303,12 @@ You can discover the available enums in each SDK at the source.
 * Flutter SDK `12.0.3`
 * [appwrite/sdk-for-flutter](https://github.com/appwrite/sdk-for-flutter/tree/dev/lib/src/enums)
 ---
+* {% only_dark %}{% icon_image src="/images/platforms/dark/react.svg" alt="React logo" size="m" /%}{% /only_dark %}
+{% only_light %}{% icon_image src="/images/platforms/react.svg" alt="React logo" size="m" /%}{% /only_light %}
+* React Native SDK `0.1.0`
+* [appwrite/sdk-for-react-native](https://github.com/appwrite/sdk-for-react-native)
+* `beta`
+---
```

Comment:
The number of columns isn't right:

<img width="720" alt="image" src="https://github.com/user-attachments/assets/426d08b1-d44d-4554-aa91-004d42bb54d4">